docs/resources/azure_express_route_circuit.md

```diff
@@ -0,0 +1,135 @@
+|--------------------------------|----------------------------------------------------------------------------------|
+| resource_group                 | Azure resource group that the targeted resource resides in. `MyResourceGroup`     |
+| circuit_name                           | Name of the circuit test. `circuit_name`                                 |
+
+Both of the parameter sets should be provided for a valid query:
+- `resource_group` and `name`
+
+## Properties
+
+| Name                           | Description                                                                      |
+|--------------------------------|----------------------------------------------------------------------------------|
+| resource_group                 | The name of the resource group in which to create the ExpressRoute circuit. Changing this forces a new resource to be created. `MyResourceGroup`    |
+| name                           | The name of the ExpressRoute circuit. Changing this forces a new resource to be created. `Myexpress circuitHostName`                          |
+| type                           | type of express ExpressRoute circuit                                                          |
+| provisioning_state             | State of express ExpressRoute circuit creation                                                |
+| location             | Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.                                              |
+| service_provider_properties_bandwidth_in_mbps             | The bandwidth in Mbps of the circuit being created on the Service Provider.                                                |
+| service_provider_properties_peering_location             | The name of the peering location and not the Azure resource location. Changing this forces a new resource to be created.                                              |
+| service_provider_properties_name             | The name of the ExpressRoute Service Provider. Changing this forces a new resource to be created.                                                |
+| service_provider_provisioning_state             | The ExpressRoute circuit provisioning state from your chosen service provider. Possible values are "NotProvisioned", "Provisioning", "Provisioned", and "Deprovisioning".                                                |
+| service_key             | State of express circuitHostName creation                                                |
+| stag             | The identifier of the circuit traffic. Outer tag for QinQ encapsulation.                                              |
+| global_reach_enabled             | ExpressRoute circuit allowGlobalReachEnable   `boolean`                                          |
+| allow_global_reach             | ExpressRoute circuit Flag denoting global reach status. `boolean`                                          |
```

Comment:
Is `allow_global_reach` a valid property?

Reply:
Yes .Ian .Sending you the response :

{
  "value": [
    {
      "name": "samir_circuit_route",
      "id": "",
      "etag": "W/\"",
      "type": "Microsoft.Network/expressRouteCircuits",
      "location": "westus2",
      "tags": {},
      "properties": {
        "provisioningState": "Updating",
        "resourceGuid": "dsd2332ds",
        "peerings": [],
        "authorizations": [],
        "serviceProviderProperties": {
          "serviceProviderName": "AARNet",
          "peeringLocation": "Sydney",
          "bandwidthInMbps": 50
        },
        "circuitProvisioningState": "Disabled",
        "allowClassicOperations": false,
        "serviceKey": "000000 3939 0000-0000-000000000000",
        "serviceProviderProvisioningState": "NotProvisioned",
        "allowGlobalReach": false,
        "globalReachEnabled": false
      },
      "sku": {
        "name": "Standard_MeteredData",
        "tier": "Standard",
        "family": "MeteredData"
      }
    }
  ]
}